### DISCUSSION TITLE
forwardRef error in jest test

### DISCUSSION THREAD

**Discussion by sandemchad** [csandman/chakra-react-select#326] [Q&A]
I am currently running into an issue where I am encountering the following error in jest tests:

>    Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object.
    Check the render method of `ForwardRef`.
    
When i downgrade to v3 of chakra-react-select everything passes but all ^4 versions appear to have this issue for me. 

Im using Chakra-ui 2.7.1 at the moment 

Have tried upgrading various packages but does not seem to be having an impact wanted to see if anyone else had any other ideas?

**Comment by csandman** [comment]
Can you give me a code sample of exactly what you're trying to do?

**Reply by sandemchad** [reply]
The jest test fails when rendering the `Select` component with or without any props ie:

The a stripped down example of the test that fails looks like this:
```javascript
import React from 'react'
import {render} from '@testing-library/react'
import {ChakraProvider, Text} from '@chakra-ui/react'
import {Select} from 'chakra-react-select'

test('it can render the select component', () => {
  render(
    <ChakraProvider>
      <Select />
    </ChakraProvider>
  )
})

test('it can render without the select component', () => {
  render(
    <ChakraProvider>
      <Text>This one renders fine</Text>
    </ChakraProvider>
  )
})
```

The test with the Select raises the error the one without doesn't
